From 27d3f533ce74410a4a2e99df92c16dbf080c7624 Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Wed, 9 Jan 2013 11:27:58 +0800
Subject: [PATCH 384/505] ENGR00239390 mx6: v4l2: Fix build warning

Fixed the warning:
mxc_v4l2_capture.c: In function 'print_pixelformat':
mxc_v4l2_capture.c:84:1: warning: no return statement in function returning
			non-void [-Wreturn-type]

mxc_v4l2_overlay.c: In function 'print_pixelformat':
mxc_v4l2_overlay.c:104:1: warning: no return statement in function returning
			non-void [-Werror=return-type]

csi_v4l2_capture.c: In function 'print_pixelformat':
csi_v4l2_capture.c:202:1: warning: no return statement in function returning
			non-void [-Wreturn-type]
csi_v4l2_capture.c: In function 'v4l_capture_test':
csi_v4l2_capture.c:287:28: warning: unused variable 'fmt' [-Wunused-variable]
csi_v4l2_capture.c: In function 'signal_thread':
csi_v4l2_capture.c:437:11: warning: variable 'err' set but not used
			[-Wunused-but-set-variable]

Signed-off-by: Robby Cai <R63905@freescale.com>
---
 test/mxc_v4l2_test/csi_v4l2_capture.c | 9 ++++-----
 test/mxc_v4l2_test/mxc_v4l2_capture.c | 4 ++--
 test/mxc_v4l2_test/mxc_v4l2_overlay.c | 4 ++--
 3 files changed, 8 insertions(+), 9 deletions(-)

diff --git a/test/mxc_v4l2_test/csi_v4l2_capture.c b/test/mxc_v4l2_test/csi_v4l2_capture.c
index 8f2c465..fe4a059 100644
--- a/test/mxc_v4l2_test/csi_v4l2_capture.c
+++ b/test/mxc_v4l2_test/csi_v4l2_capture.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2009-2012 Freescale Semiconductor, Inc. All rights reserved.
+ * Copyright 2009-2013 Freescale Semiconductor, Inc. All rights reserved.
  */
 
 /*
@@ -192,7 +192,7 @@ static int find_video_device(void)
 		return fd_v4l;
 }
 
-static int print_pixelformat(char *prefix, int val)
+static void print_pixelformat(char *prefix, int val)
 {
 	printf("%s: %c%c%c%c\n", prefix ? prefix : "pixelformat",
 					val & 0xff,
@@ -284,7 +284,6 @@ int v4l_capture_test(int fd_v4l)
 {
 	struct fb_var_screeninfo var;
         struct v4l2_buffer buf;
-        struct v4l2_format fmt;
 	char fb_device[100] = "/dev/fb0";
 	int fd_fb = 0;
 	int frame_num = 0, i, fb0_size;
@@ -435,12 +434,12 @@ int process_cmdline(int argc, char **argv)
 
 static int signal_thread(void *arg)
 {
-	int sig, err;
+	int sig;
 
 	pthread_sigmask(SIG_BLOCK, &sigset, NULL);
 
 	while (1) {
-		err = sigwait(&sigset, &sig);
+		sigwait(&sigset, &sig);
 		if (sig == SIGINT) {
 			printf("Ctrl-C received. Exiting.\n");
 		} else {
diff --git a/test/mxc_v4l2_test/mxc_v4l2_capture.c b/test/mxc_v4l2_test/mxc_v4l2_capture.c
index 81107c9..8b60aea 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_capture.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_capture.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2012 Freescale Semiconductor, Inc. All rights reserved.
+ * Copyright 2004-2013 Freescale Semiconductor, Inc. All rights reserved.
  */
 
 /*
@@ -74,7 +74,7 @@ int g_extra_pixel = 0;
 int g_capture_mode = 0;
 char g_v4l_device[100] = "/dev/video0";
 
-static int print_pixelformat(char *prefix, int val)
+static void print_pixelformat(char *prefix, int val)
 {
 	printf("%s: %c%c%c%c\n", prefix ? prefix : "pixelformat",
 					val & 0xff,
diff --git a/test/mxc_v4l2_test/mxc_v4l2_overlay.c b/test/mxc_v4l2_test/mxc_v4l2_overlay.c
index 71c056f..1c8e491 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_overlay.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_overlay.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2012 Freescale Semiconductor, Inc. All rights reserved.
+ * Copyright 2004-2013 Freescale Semiconductor, Inc. All rights reserved.
  */
 
 /*
@@ -94,7 +94,7 @@ int alpha_buf_size = 0;
 int ctrl_c_rev = 0;
 int g_fd_fb_fg = 0;
 
-static int print_pixelformat(char *prefix, int val)
+static void print_pixelformat(char *prefix, int val)
 {
 	printf("%s: %c%c%c%c\n", prefix ? prefix : "pixelformat",
 					val & 0xff,
-- 
1.8.0

